app/javascript/listings/listings.jsx

```diff
@@ -310,6 +342,9 @@ export class Listings extends Component {
             className="tag-close"
             onClick={e => this.removeTag(e, tag)}
             data-no-instant
+            role="button"
+            onKeyPress={e => this.removeTag(e, tag)}
```

Comment:
cool!

Reply:
removeTag should be called when the user presses enter:

`e.key === 'Enter'`